### ISSUE TITLE
can not use

### ISSUE THREAD

**Issue by happyfsyy** [gkonovalov/android-vad#12]
Hello,
I imported this package, but I couldn't use it. 
The error message is: Caused by: java.lang.IllegalArgumentException: Error can't init WebRTC VAD!! 

The error code is as follows:
```kotlin
mVad=Vad.builder()
    .setModel(Model.WEB_RTC_GMM)
    .setSampleRate(SampleRate.SAMPLE_RATE_8K)
    .setFrameSize(FrameSize.FRAME_SIZE_160)
    .setMode(Mode.VERY_AGGRESSIVE)
    .build()
```

**Comment by abb128** [comment]
May be related to #11?

**Comment by gkonovalov** [comment]
Hi @happyfsyy  
Thank you so much for your comment! I have just released version 2.0.3, which includes a bug fix. I wanted to inform you that as part of this release, I have split the library into smaller modules. This means that we now have separate dependencies for each module.

`dependencies {
    implementation 'com.github.gkonovalov.android-vad:webrtc:2.0.3'
}`
